staging-service/src/test/java/com/github/cafdataprocessing/services/staging/healthcheck/test/DiskSpaceHealthIndicatorWithTimeoutTest.java

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright 2019-2022 Micro Focus or one of its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.github.cafdataprocessing.services.staging.healthcheck.test;
+
+import com.github.cafdataprocessing.services.staging.StagingController;
+import com.github.cafdataprocessing.services.staging.StagingProperties;
+import com.github.cafdataprocessing.services.staging.dao.BatchDao;
+import com.github.cafdataprocessing.services.staging.models.StatusResponse;
+import java.io.File;
+import java.io.IOException;
+import javax.servlet.http.HttpServletRequest;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
+import org.mockito.Mockito;
+import org.springframework.boot.actuate.autoconfigure.system.DiskSpaceHealthIndicatorProperties;
+import org.springframework.http.ResponseEntity;
+import org.springframework.util.unit.DataSize;
+
+/**
+ *
+ * @author TBroadbent
+ */
+public class DiskSpaceHealthIndicatorWithTimeoutTest
+{
+    private StagingController controller;
+
+    BatchDao fileSystemDao;
+    HttpServletRequest request;
+    StagingProperties stagingProperties;
+    File file;
```

Comment:
Make private, and File not needed

Reply:
I'm not sure these can be made final due to being initialized in setup, but can be private
![image](https://user-images.githubusercontent.com/96247486/170283974-94aee2bf-be7c-4898-9919-f94b7c1e5a4b.png)

Reply:
Correct, suggested change was just make private for these.